scenarioo-client/app/manage/label-colors/label-configuration.ts

```diff
@@ -0,0 +1,32 @@
+import {IFlatLabelConfiguration} from '../../generated-types/backend-types';
+
+export class LabelConfiguration implements IFlatLabelConfiguration {
+
+    backgroundColor: string;
+    foregroundColor: string;
+    name: string;
+
+    private constructor(backgroundColor: string, foregroundColor: string, name: string) {
```

Comment:
just add the public keyword in front of every parameter and you will have your public fields including the assignment

Reply:
Thanks for the tip! Will do that